src/coreclr/pal/tests/palsuite/file_io/GetSystemTime/test1/test.cpp

```diff
@@ -100,24 +95,24 @@ PALTEST(file_io_GetSystemTime_test1_paltest_getsystemtime_test1, "file_io/GetSys
             "and it is currently showing %d.",TheTime.wMilliseconds);
     }
 
-    /* check if two consecutive calls to system time return */
-    /* correct time in ms after sleep() call. */
-    for (i = 0; i<5 ;i++)
-    { 
+    /* verify that two consecutive calls to system time return */
+    /* different values of seconds across sleep() call. */
+    /* do this 5 times in case we are super unlucky. */
+    float avgDeltaFileTime = 0;
+    for (int i = 0; i < 5; i++)
+    {
         GetSystemTime(&firstTime);
         Sleep(1000);
-        GetSystemTime(&secondTime);    
-        avgDeltaFileTime +=  abs(firstTime.wSecond - secondTime.wSecond );
-
-    }     
+        GetSystemTime(&secondTime);
+        avgDeltaFileTime += abs(firstTime.wSecond - secondTime.wSecond);
+    }
 
-    if( (avgDeltaFileTime/5) < 1.0)
+    if( (avgDeltaFileTime / 5) == 0)
```

Comment:
Would it be better to change this condition to `<= 0`? (since this test is known to be passing on several platforms for a long time)